content/en/blog/2023/waypoint-proxy-made-simple/index.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Istio Ambient Waypoint Proxy Made Simple"
+description: Introducing the new destination oriented waypoint proxy for simplicity and scalability.
+publishdate: 2023-03-31
+attribution: "Lin Sun (Solo.io), John Howard (Google)"
+keywords: [istio,ambient,waypoint]
+---
+
+Ambient splits Istio’s functionality into two distinct layers, secure overlay layer and
+Layer 7 processing layer. The waypoint proxy is an optional component that is Envoy-based
+and handles L7 processing for workloads it manages. Since the [initial ambient launch](/blog/2022/introducing-ambient-mesh/) in 2022,
+we have made significant changes to simplify waypoint configuration, debuggability and scalability.
+
+## Architecture of waypoint
```

Comment:
My thought is we want to use waypoint proxy, and not just waypoint.
```suggestion
## Architecture of waypoint proxies
```
or maybe Waypoint proxy architecture